runtime/src/main/java/com/xatkit/plugins/moodle/platform/utils/MessageObject.java

```diff
@@ -39,7 +39,7 @@ public MessageObject() {
      */
     public MessageObject(String message, String username) {
```

Comment:
Argument should be named `userId` instead of `username`

Reply:
resolved in next commit